Documentation/docs/learn/python_quick_start.md

```diff
@@ -0,0 +1,495 @@
+
+These file formats can also be used with native ITK Python.
+
+Install the binary Python package:
+
+```sh
+pip install itk-webassemblyinterface
+```
+
+Then use with `itk.imread`, `itk.imwrite`, `itk.meshread`, `itk.meshwrite`. Example:
+
+```py
+import itk
+
+image = itk.imread('cthead1.png')
+itk.imwrite(image, 'cthead1.iwi')
+itk.imwrite(image, 'cthead1.iwi.cbor')
+
+mesh = itk.meshread('cow.vtk')
+itk.meshwrite(mesh, 'cow.iwm')
+itk.meshwrite(mesh, 'cow.iwm.cbor')
+```
+
+### ITK and Xarray
+
+An `itk.Image` can be converted to and from an [xarray.DataArray](https://xarray.pydata.org/en/stable/generated/xarray.DataArray.html) while
+preserving metadata:
+
+```py
+da = itk.xarray_from_image(image)
+
+image = itk.image_from_xarray(da)
+```
+
+### ITK and VTK
+
+An `itk.Image` can be converted to and from a [vtk.vtkImageData](https://vtk.org/doc/nightly/html/classvtkImageData.html) while
+preserving metadata:
+
+```py
+vtk_image = itk.vtk_image_from_image(image)
+
+image = itk.image_from_vtk_image(vtk_image)
+```
+
+### ITK and napari
+
+An `itk.Image` can be converted to and from a [napari.layers.Image](https://napari.org/stable/api/napari.layers.Image.html#napari.layers.Image) while
+preserving metadata with the [itk-napari-conversion package](https://github.com/InsightSoftwareConsortium/itk-napari-conversion).
+
+### ITK Python types
+
+| C++ type             | Python type          | NumPy dtype  |
+| -------------------- | -------------------- | ------------ |
+| float                | itk.F                | np.float32   |
+| double               | itk.D                | np.float64   |
+| unsigned char        | itk.UC               | np.uint8     |
+| std::complex\<float> | itk.complex\[itk.F\] | np.complex64 |
+
+This list is not exhaustive and is only presented to illustrate the type names. The complete list of types can be found in the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf).
+
+Types can also be obtained from their name in the C programming language:
+
+```py
+itk.F == itk.ctype('float') # True
+```
+
+To cast the pixel type of an image, use `.astype`:
+
+```py
+image = itk.imread(input_filename)
+
+# Cast to an unsigned char pixel type
+cast_image = image.astype(itk.UC)
+
+# Equivalent
+cast_image = image.astype(np.uint8)
+
+itk.imwrite(cast_image, output_filename)
+```
+
+### Metadata dictionary
+
+An `itk.Image` has a metadata dict of `key: value` pairs.
+
+The metadata dictionary can be retrieved with:
+
+```
+meta_dict = dict(image)
+```
+
+For example:
+
+```
+In [3]: dict(image)
+Out[3]:
+{'0008|0005': 'ISO IR 100',
+ '0008|0008': 'ORIGINAL\\PRIMARY\\AXIAL',
+ '0008|0016': '1.2.840.10008.5.1.4.1.1.2',
+ '0008|0018': '1.3.12.2.1107.5.8.99.484849.834848.79844848.2001082217554549',
+ '0008|0020': '20010822',
+```
+
+Individual dictionary items can be accessed or assigned:
+
+```
+print(image['0008|0008'])
```

Comment:
Do we use simple ticks or double ticks? black uses double ticks by default if I am not mistaken.

Reply:
Ok, switched to double.